graylog2-server/src/main/java/org/graylog2/plugin/inputs/Extractor.java

```diff
@@ -281,7 +287,14 @@ private void runConverters(Message msg) {
                     }
                 } catch (Exception e) {
                     this.converterExceptions.incrementAndGet();
-                    LOG.error("Could not apply converter [" + converter.getType() + "] of extractor [" + getId() + "].", e);
+                    final String error = "Could not apply converter [" + converter.getType() + "] of extractor <" + getTitle() + " (" + getId() + ")>";
+                    if (LOG.isDebugEnabled()) {
+                        LOG.error(error, e);
+                    } else {
+                        LOG.error("{}:\n{}", error, ExceptionUtils.getShortenedStackTrace(e));
```

Comment:
The logging framework must support a special formatting symbol for line breaks making it OS-independent

Reply:
Not that I know of.
In a previous version I had
```suggestion
                        LOG.error("{}:{}{}", error, System.lineSeparator(), ExceptionUtils.getShortenedStackTrace(e));
```
But it felt over-engineered because we only run on unix-ish platforms anyway